test/javascript/components/mentoring/mentor_conversations_list/pagination.test.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom/extend-expect'
+import Pagination from '../../../../../app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx'
+
+test('clicking on "First" sets page to 1', () => {
```

Comment:
If you add the `aria` values, it is nice to _at least_ test the "current" (for announcing) and that the labels have more than just the page number.